src/pages/weather-physical-description/weather-physical-description.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams } from 'ionic-angular';
+import { DashboardPage } from '../dashboard/dashboard';
+
+/**
+ * Generated class for the WeatherPhysicalDescriptionPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@Component({
+  selector: 'page-weather-physical-description',
+  templateUrl: 'weather-physical-description.html',
+})
+export class WeatherPhysicalDescriptionPage {
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {
+  }
+
+  ionViewDidLoad() {
+    console.log('ionViewDidLoad WeatherPhysicalDescriptionPage');
+  }
+
+  goToDashboard() {
+    this.navCtrl.push(DashboardPage);
```

Comment:
I know that it's a sample, not production ready code but I think it's worth to mention that when you popping back you shouldn't push page onto a stack. Rather you should pop it from the stack.

Reply:
I think we should be treating this as production ready so I'll change this to pop